### ISSUE TITLE
Unable to download this depot

### ISSUE THREAD

**Issue by nisehime** [SteamRE/DepotDownloader#349]
Hello. I found depot 1523211. When I tried to download (actually I thought `-manifest-only` would give file list) I get this error:

```
DepotDownloader.exe -app 1030830 -depot 1523211 -username *** -manifest-only

Enter account password for "***": **************
Connecting to Steam3... Done!
Logging '***' into Steam3...Disconnected from Steam
This account is protected by Steam Guard.
Please enter your 2 factor auth code from your authenticator app: ***
Retrying Steam3 connection... Done!
Logging '***' into Steam3... Done!
Using Steam3 suggested CellID: 7
Got 94 licenses for account!
Got session token!
Accepted new login key for account ***
Got AppInfo for 1030830
Using app branch: 'Public'.
Got depot key for 1523211 result: Blocked
Disconnected from Steam
No valid depot key for 1523211, unable to download.
Total downloaded: 0 bytes (0 bytes uncompressed) from 0 depots
```

However, I was able to do this with steamcmd

```
Steam>download_depot 1030830 1523211
Downloading depot 1523211 (99 MB) ...
Depot download complete : "***\steamcmd\steamapps\content\app_1030830\depot_1523211" (4 files, manifest 4005512837758401031)
```
It's 2K launcher.

So this depot is accesible, but steamdb doesn't have any info about it, except referenced packages. ~~Might this be related?~~ Actually, it displays manifests changes, but only on the app page for some reason. I don't understand.

**Comment by xPaw** [comment]
https://github.com/SteamRE/DepotDownloader/blob/72683ef35cc60d6f55f6e79d2902ac564b0c2b7d/DepotDownloader/ContentDownloader.cs#L654-L655

I think this code shouldn't apply when requesting depot keys.

**Comment by xPaw** [comment]
Yep I tested this, for GetDepotDecryptionKey it has to be the non-depotfromapp appid, because if the you don't own depotfromapp, you can't get the depot key.

And for GetManifestRequestCode it doesn't matter, it worked for both appids. In fact, it seems to work for *any* appid. I even tried a non existing appid, and it still worked.

**Comment by azuisleet** [comment]
This has been fixed in 3143362d83b0f6753e48b50fa7e06735756b2361 and released in 2.4.7

https://github.com/SteamRE/DepotDownloader/releases/tag/DepotDownloader_2.4.7
